.codecov.yml

```diff
@@ -0,0 +1,17 @@
+ignore:
+  - "skimage/external/setup.py"
+  - "skimage/_build.py"
+  - "scripts/skivi.py"
+  - "setup.py"
+  - "**/setup.py"
+
+coverage:
+  status:
+    project:
+      default:
+        # Allow the coverage to drop by 0.1% even when posting success
+        # Drops on the order 0.01% are typical even when no change occurs
```

Comment:
Doesn't match still :smile: .

Reply:
@soupault yes it does. He's saying codecov can report 0.01% drop even when nothing has changed, so we set the tolerance to 0.1%.

Reply:
@soupault not sure if the rewording helped. 0.1 and 0.01 look like typos ;)